lighthouse-cli/bin.ts

```diff
@@ -209,7 +216,7 @@ function getDebuggableChrome(flags: {skipAutolaunch: boolean, port: number, sele
                                chromeFlags: string}): Promise<ChromeLauncher> {
   const chromeLauncher = new ChromeLauncher({
     port: flags.port,
-    additionalFlags: flags.chromeFlags.split(' '),
+    chromeFlags: flags.chromeFlags.split(' '),
```

Comment:
this is a breaking change and some other folks rely on the API, but @samccone's work here will break them too and hopefully have some docs so its easy to do the right thing.

all that said, +1 to this change.